src/signals/settings/users/containers/Overview/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row, Column } from '@datapunt/asc-ui';
+
+import LoadingIndicator from 'shared/components/LoadingIndicator';
+import ListComponent from 'components/List';
+import Pager from 'components/Pager';
+
+import PageHeader from 'signals/settings/components/PageHeader';
+import useFetchUsers from './hooks/useFetchUsers';
+
+
+const UsersOverview = ({ page }) => {
+  const { isLoading, users } = useFetchUsers();
+
+  return (
+    <div className="users-overview-page">
+      <PageHeader title={`Gebruikers (${users.length})`} />
+
+      <Row>
+        <Column span={12} wrap>
+          <Column span={12}>
+            {isLoading ? (
```

Comment:
Same as earlier. 
Do we need the `<Columns>` when we're still loading? Or can we move them inside the `loading` check?